Project3_Bulls_and_Cows/Game.py

```diff
@@ -0,0 +1,75 @@
+import random
+import itertools
+
+def game_header():
+    print('Hi there!')
+    print("I've generated a random 4 digit number for you.")
+    print("Let's play a bulls and cows game.")
+
+
+def number_generator():
+    generated_number = [int(x) for x in range(10)]
+    secret = []
+
+    for i in generated_number:
+        number = random.choice(generated_number)
+        if len(secret) == 4:
+            break
+        else:
+            secret.append(number)
+            generated_number.remove(number)
+    return secret
+
+
+def ask_for_number():
+    while True:
+        a = input('Enter a number:')
+        try:
+            a = int(a)
+        except ValueError:
+            print('Please enter number only.')
+            continue
+        if a not in range(1000, 9999):
```

Comment:
This still checks that a user enter a number larger than `1000` but secret can now start with `0`. Good luck with guessing `0123` because you won't be able to input the number.